Library/Formula/anttweakbar.rb

```diff
@@ -64,3 +63,16 @@ def install
          }
          if (gl_dyld)
          {
+diff --git a/src/Makefile.osx b/src/Makefile.osx
```

Comment:
What's the best way to submit just this patch to the upstream developers (I think that's what you requested)?